interfaces/cython/cantera/utils.pyx

```diff
@@ -31,6 +31,10 @@ def add_directory(directory):
     """ Add a directory to search for Cantera data files. """
     CxxAddDirectory(stringify(directory))
 
+def get_data_directories(sep=':'):
```

Comment:
I think you need to use `os.pathsep` instead of `':'` here, otherwise this will fail badly on Windows.